src/client/components/pages/entities/related-collections.js

```diff
@@ -29,15 +29,21 @@ function EntityRelatedCollections({collections}) {
 		<Row>
 			<Col>
 				<h2>Related Collections</h2>
-				{collections &&
-				<ul className="list-unstyled">
-					{collections.map((collection) => (
-						<li key={collection.id}>
-							<a href={`/collection/${collection.id}`}>{collection.name}</a> by {' '}
-							<a href={`/editor/${collection.ownerId}`}>{collection.owner.name}</a>
-						</li>
-					))}
-				</ul>
+				{collections && collections.length > 0 ? (
+					<ul className="list-unstyled">
+						{collections.map((collection) => (
+							<li key={collection.id}>
+								<a href={`/collection/${collection.id}`}>{collection.name}</a> by {' '}
+								<a href={`/editor/${collection.ownerId}`}>{collection.owner.name}</a>
+							</li>
+						))}
+					</ul>
+				) :
+					<p className="text-muted">
+						<b>No collections found.</b>
```

Comment:
What do you think about this?
```suggestion
						<b>This entity does not appear in any public collection.</b>
```

Reply:
This is more descriptive but consistent with the theme and language of this website, so sounds good 👍.